config/samples/core/kustomization.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  sample: "true"
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+resources:
+- goharbor_v1alpha2_core.yaml
+
+secretGenerator:
+- name: core-secret
+  literals:
+  - secret=unsecure-core-secret
+  type: "goharbor.io/single-secret"
+- name: csrf-key
+  literals:
+  - key=12345678901234567890123456789012
+  type: "goharbor.io/csrf"
+- name: admin-core-secret
+  literals:
+  - secret=Harbor12345
+  type: "goharbor.io/single-secret"
+- name: core-database-encryption
+  literals:
+    - secret=1234567890123456
+  type: "goharbor.io/single-secret"
+- name: jobservice-secret
+  literals:
+  - secret=unsecure-jobservice-secret
+  type: "goharbor.io/single-secret"
+- name: core-token-certificate
```

Comment:
This should probably be replaced by an `Issuer` and a `Certificate`.
#60

Reply:
Any more details about fixing the problem of #60?